Source/Fuse.Elements/Caching/ElementAtlas.uno

```diff
@@ -167,8 +159,12 @@ namespace Fuse.Elements
 			_framebuffer.Unpin();
 		}
 
-		void FillFramebuffer(DrawContext dc, bool drawAll, Rect scissorRectInClipSpace)
+		void FillFramebuffer(DrawContext dc, framebuffer fb, bool drawAll, Rect scissorRectInClipSpace)
 		{
+			// Changing the framebuffer is expensive. This loop actually doesn't draw anything in
+			// common case. Don't push the render target unless we actually have to render something.
```

Comment:
If this gets here without drawing anything, I think something higher level failed. So I think this is fixing the symptom, rather than the actual issue. The `_invalidElements > 0`-check around the calls to FillFramebuffer should have avoided this case.